StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift

```diff
@@ -54,6 +56,13 @@ extension TextFieldElement {
                 }
             }
 
+            // If this is coming from the LastFourConfiguration, cardBrand(for: text) will retrieve a card brand from •••• •••• •••• last4, which may be incorrect, so we pass in the card brand for that case
+            if let cardBrand = cardBrand,
+               cardBrandDropDown == nil {
+                rotatingCardBrandsView.cardBrands = [cardBrand]
+                return rotatingCardBrandsView
+            }
```

Comment:
Any reason we can't have this below on line 77.
`rotatingCardBrandsView.cardBrands = [overrideCardBrand ?? cardBrand]`

Reply:
cardBrand(for: text) takes the text in the text field (•••• •••• •••• last4) and sanitizes it, making it just the last4, and it tries to figure out what the card brand is from there. It could potentially return an unknown, so we can't guarantee it'll go into the else.